webcompat/static/js/lib/issue-list.js

```diff
@@ -299,56 +307,20 @@ issueList.IssueView = Backbone.View.extend(
     loadIssues: function() {
     // Attemps to load model state from URL params, if present,
     // otherwise grab model defaults and load issues
-      var category;
-      var queryMatch;
-
       this._urlParams = location.search.slice(1);
-      var urlParams = this._urlParams;
 
     // There are some params in the URL
-      if (urlParams.length !== 0) {
-        queryMatch = urlParams.match(this._searchRegex);
-        if (!this._isLoggedIn && queryMatch) {
-        // We're dealing with an un-authed user, with a q param.
-          this.doGitHubSearch(urlParams);
-          this.updateModelParams(urlParams);
-          _.delay(function() {
-          // TODO: update search input from query param for authed users.
-            issueList.events.trigger('search:update', queryMatch[1]);
-          }, 0);
-        } else if (category = window.location.search.match(this._filterRegex)) {
-        // If there was a stage filter match, fire an event which loads results
-          this.updateModelParams(urlParams);
-          _.delay(function() {
-            issueList.events.trigger('filter:activate', category[1]);
-          }, 0);
-        } else {
-        // Only bother to update/merge model params if we're not loading the defaults
-          if (urlParams !== $.param(this.issues.params)) {
-            this.updateModelParams(urlParams);
-          }
+      if (this._urlParams.length !== 0) {
```

Comment:
:clock430:   (just leaving a bookmark for myself, I'll pick up review here tomorrow) :clock8: